src/test/e2e/tests/specs.test.ts

```diff
@@ -176,6 +176,43 @@ describe('Finding a court or tribunal listing', () => {
     });
   });
 
```

Comment:
this flow won't work as this is supposed to be restricted page and as such should be within media Media User Login describe (notice on line 246 and 270 there are different branching for each radio select

Reply:
add the mocksession to test.